plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayPublisherExtension.kt

```diff
@@ -23,29 +27,28 @@ open class PlayPublisherExtension : AccountConfig by PlayAccountConfigExtension(
      * https://github.com/Triple-T/gradle-play-publisher#untrack-conflicting-versions. Disabled by
      * default.
      */
+    @get:Input
     var untrackOld = false
     /**
      * Specify the initial user percent intended to receive a 'rollout' update (see [track]).
      * Default is 10% == 0.1.
      */
+    @get:Input
     var userFraction = 0.1
-
-    /**
-     * Choose whether or not to upload images when publishing the Play Store listing. Disabled by
-     * default for performance reasons.
-     */
-    var uploadImages = false
```

Comment:
Let's create a doc issue for this change because we need to mention this in migration hints. People have to remove this property from their gradle files

Reply:
Done: #315